### PR TITLE
Add document for setting filecoindot

### LINKED ISSUE #63: Add document for filecoindot-ocw
awesome! 
this is great progress.
One nit, could you please add some docs to the lib.rs, how the rpc api interacts with the storage and pallet, basically how everything works? 
either in this PR or follow up

_Originally posted by @mattsse in https://github.com/ChainSafe/filecoindot/pull/60#pullrequestreview-793737032_

### PR DESCRIPTION
**Pull request by clearloop**
## Changes

- based on #68
- [x] add docs in `/README.md`
  - [x] configure runtime
  - [x] add signer account
  - [x] set rpc endpoint

## Tests

```

```

## Issues

- Closes #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,170 @@ framework, including Polkadot parachains.
...
+Here you need to register filecoindot and the offchain worker's logic to your `runtime.rs`:
+
+```rust
+// runtime.rs
+
+parameter_types! {
+    pub const OffchainWorkerTimeout: u64 = 1_000_000;
+}
+
+// ManagerOrigin as root
```
Comment (mattsse) [inline README.md:30]:
```suggestion
// ManagerOrigin which manages the approved relayer set, in this case root
```

File: README.md [hunk 2]
```diff
@@ -10,7 +10,170 @@ framework, including Polkadot parachains.
...
+    type WeightInfo = ();
+    type AuthorityId = filecoindot::FilecoindotId;
+    type OffchainWorkerTimeout = OffchainWorkerTimeout;
+}
+
+// For pallet-example-offchain-worker
+parameter_types! {
+    pub const GracePeriod: BlockNumber = 3;
+    pub const UnsignedInterval: BlockNumber = 3;
+    pub const UnsignedPriority: BlockNumber = 3;
```
Comment (mattsse) [inline README.md:45]: add brief explainer

File: README.md [hunk 3]
```diff
@@ -10,7 +10,170 @@ framework, including Polkadot parachains.
...
+        let tip = 0;
+        let extra: SignedExtra = (
+            frame_system::CheckSpecVersion::<Runtime>::new(),
+            frame_system::CheckTxVersion::<Runtime>::new(),
+            frame_system::CheckGenesis::<Runtime>::new(),
+            frame_system::CheckEra::<Runtime>::from(generic::Era::mortal(period, current_block)),
+            frame_system::CheckNonce::<Runtime>::from(index),
+            frame_system::CheckWeight::<Runtime>::new(),
+            pallet_transaction_payment::ChargeTransactionPayment::<Runtime>::from(tip),
+        );
```
Comment (mattsse) [inline README.md:74]: this looks pretty verbose,

can we either,

* exclude this from the overview?
* explain whats happening/why we need this?
* link to existing offchain worker docs?

File: README.md [hunk 4]
```diff
@@ -10,7 +10,170 @@ framework, including Polkadot parachains.
...
+{
+    type OverarchingCall = Call;
+    type Extrinsic = UncheckedExtrinsic;
+}
+```
+
+### 1. set signer account to your node
+
+Here we need to generate an account for our offchain worker first
+
```
Comment (mattsse) [inline README.md:108]: add link to the subkey command

File: README.md [hunk 5]
```diff
@@ -10,7 +10,170 @@ framework, including Polkadot parachains.
...
+2021-10-31 21:58:57 〽️ Prometheus exporter started at 127.0.0.1:9615
+2021-10-31 21:58:57 Listening for new connections on 127.0.0.1:9944.
+2021-10-31 21:59:00 🙌 Starting consensus session on top of parent 0x6b98f8856f41abfb45da878188aaff1419c1cb6bbb8a56a346e9f1f39613a1a4
+2021-10-31 21:59:00 🎁 Prepared block for proposing at 169 [hash: 0x9f3941c44fdacc7f240b0471c97cd61e5010377619501285999ab0ff4d0edbbc; parent_hash: 0x6b98…a1a4; extrinsics (1): [0x1dd1…1b8b]]
+2021-10-31 21:59:00 🔖 Pre-sealed block for proposal at 169. Hash now 0x53480ec65b6b2c13126a96c62089af0244a570ba961efdd0d377a44df12a5a5b, previously 0x9f3941c44fdacc7f240b0471c97cd61e5010377619501285999ab0ff4d0edbbc.
+2021-10-31 21:59:00 ✨ Imported #169 (0x5348…5a5b)
+2021-10-31 21:59:00 bootstrap filecoindot ocw with filecoin rpc endpoint http://user:pass@infura.io
+```
+
+the filecoindot offchain work has been set up!
```
Comment (mattsse) [inline README.md:173]:
```suggestion
the filecoindot offchain worker has been set up!
```

**Review by mattsse — CHANGES_REQUESTED** [review]
some nits,
can we move the ocw demo into the ocw crate otherwise the main readme is kinda bloated?
Instead, we reference from the root readme

**Comment by clearloop** [comment]
This doc missed the `set_relayer` extrinsic, should we doc it with `polkadot.js` or just a brief introduce ( if we serve rust doc in the CI, we can reference it)?

**Review by mattsse — APPROVED** [review]
lgtm

**Comment by mattsse** [comment]
> This doc missed the `set_relayer` extrinsic, should we doc it with `polkadot.js` or just leave a brief introduce ( if we serve rust doc in the CI, we can reference it)?

I think the pallet docs are sufficient here, set_relayer should be self-explanatory.

**Pull request merged**